### PR TITLE
added fast scan for nmap

### PR DESCRIPTION
**Pull request by prankymat**
Faster nmap scanning options

### CODE REVIEW

File: pages/common/nmap.md [hunk 1]
```diff
@@ -9,3 +9,11 @@
 - discover hosts in the 192.168.0.X area (no port scan)
 
 `nmap -sn {{192.168.0.1/24}}`
+
+- faster scan of a single host (scans for common ports)
+
+`nmap {{192.168.0.1}}`
```
Comment (waldyrious) [inline pages/common/nmap.md]: This is identical to the first example. Are you sure you didn't forget something?

Comment (prankymat) [inline pages/common/nmap.md]: Sorry! Forgot to add the tag! Will fix.

**Comment by prankymat** [comment]
Fixed issue. Thanks

File: pages/common/nmap.md (hunk 1 above)
Comment (prankymat) [inline pages/common/nmap.md]: Fixed with latest commit

**Comment by waldyrious** [comment]
cool, LGTM :+1:

**Comment by felixonmars** [comment]
Please squash the commits, thanks :)

**Comment by prankymat** [comment]
Will squash commits! Thanks

**Comment by ostera** [comment]
@prankymat, so it seems your branch has gone bogus. Let's try to fix it with `git reflog`! What we want to do is go back to the point where your branch was adding only your commit.

Doing `git reflog` should give you a list of things you've done to your repo, not unlike this:

```
42ff891 HEAD@{0}: checkout: moving from master to fast-nmap
e941175 HEAD@{1}: clone: from https://github.com/tldr-pages/tldr
```

When you find the line where your commit was added, reset your branch with this: `git reset --hard <hash>`, where `<hash>` is the hash of your commit.

Right afterwards you can and should pull the latest master, and rebase your branch on top of it:

```
git remote add upstream https://github.com/tldr-pages/tldr
git pull upstream master
git checkout fast-nmap
git rebase upstream/master
```

After that you should be able to force push your branch, and this PR should be updated.

Good luck! :ok_hand:

**Comment by prankymat** [comment]
@ostera thanks so much for your guide!!! I've successfully squashed my commits into one now :) :+1:

**Comment by waldyrious** [comment]
Merging this seems at odds with what was being discussed in #451. @prankymat please comment there to add to the discussion :)

**Comment by prankymat** [comment]
:+1: to @waldyrious. Should not merge this request at this point until discussion in #451 has done.

**Pull request merged**